src/components/WebexActivityStream/Greeting.jsx

```diff
@@ -1,159 +1,42 @@
-              id="Stroke-19"
-            />
-            <path
-              d="M6.18107692,17.8535385 C7.678,18.4735385 9.02876923,18.952 10.6195385,19.2889231 L12.0533846,15.7581538 L12.138,19.7458462 C16.2349231,20.1181538 21.898,19.4212308 25.9641538,17.7381538 L26.0103077,17.7381538 C27.0918462,19.4996923 27.7149231,21.572 27.7149231,23.7904615 C27.7149231,30.1935385 22.5241538,35.3843077 16.1210769,35.3843077 C9.718,35.3843077 4.52723077,30.1935385 4.52723077,23.7904615 C4.52723077,21.7458462 5.05646154,19.8243077 5.98569231,18.1566154 L6.18107692,17.8535385 Z"
-              id="Stroke-21"
-              strokeLinecap="round"
-              strokeLinejoin="round"
-            />
-            <path
-              d="M0.000153846167,29.5121538 C0.000153846167,45.8106154 13.2124615,59.0244615 29.5124615,59.0244615 C45.8109231,59.0244615 59.0247692,45.8106154 59.0247692,29.5121538"
-              id="Stroke-23"
-            />
-            <path
-              d="M39.0553846,14.1330769 C42.1815385,18.3853846 47.8369231,20.27 53.0507692,18.3823077 C54.4338462,17.8823077 55.6723077,17.1561538 56.7369231,16.2623077"
-              id="Stroke-25"
-              strokeLinecap="round"
-              strokeLinejoin="round"
-            />
-          </g>
-        </g>
-      </g>
-    </svg>
-  );
-}
+import {PHONE_LARGE} from '../breakpoints';
+import {useElementDimensions} from '../hooks';
 
 /**
  * Greeting displayed on a new/empty room.
  *
  * @param {object} props  Data passed to the component
  * @param {string} props.className  Custom CSS class to apply
- * @param {string} props.personName  Name of the person to greet
  * @param {object} props.style  Custom style to apply
  * @returns {object} JSX of the component
  */
-export default function Greeting({className, personName, style}) {
-  let svg = <GreetingSpaceSVG />;
-  let description = 'This is a shared space between you and other group members. Here\'s where you\'ll see shared messages, files, and a call history with this space.';
-
-  const [cssClasses, sc] = webexComponentClasses('greeting', className);
-
-  if (personName) {
-    svg = <GreetingDirectSVG />;
-    description = `This is your private conversation with ${personName}. Here's where you'll see shared messages, files, and a call history with this person.`;
-  }
+export default function Greeting({className, style}) {
+  const greetingRef = useRef();
+  const {width} = useElementDimensions(greetingRef);
+  const [cssClasses, sc] = webexComponentClasses('greeting', className, {
+    mobile: width <= PHONE_LARGE,
+  });
 
   return (
-    <div className={cssClasses} style={style}>
+    <div ref={greetingRef} className={cssClasses} style={style}>
       <div className={sc('header')}>
-        {svg}
-        <div className={sc('description')}>{description}</div>
+        <div className={sc('title')}>
+          Let’s start chatting
+          {' '}
```

Comment:
What's this?

Reply:
Not sure, but this was a eslint error fix, it could be a white space fix